doc/source/install.rst

```diff
@@ -202,15 +202,15 @@ pandas is equipped with an exhaustive set of unit tests, covering about 97% of
 the code base as of this writing. To run it on your machine to verify that
 everything is working (and that you have all of the dependencies, soft and hard,
 installed), make sure you have `pytest
-<http://doc.pytest.org/en/latest/>`__ and run:
+<http://doc.pytest.org/en/latest/>`__ >= 3.6 and run:
```

Comment:
Minor comment:  The URL redirects me from `doc.pytest` to <code>doc<b><i>s</i></b>.pytest</code>; would be nice to use the newer URL in case the redirect gets removed.